R/calcValidYield.R

```diff
@@ -42,20 +42,26 @@ calcValidYield  <-  function(datasource = "FAO", future = NULL, physical = TRUE)
 
     if (!is.null(future)) stop("Future options is not available for source type 'FAO'.")
 
-    past <- findset("past")
-
     # Calculate areas of individual crops and pasture
     croparea  <-  collapseNames(calcOutput("Croparea", sectoral = "kcr",
-                                           physical = physical, aggregate = FALSE)[, past, ])
+                                           physical = TRUE, aggregate = FALSE))
```

Comment:
Should we really ignore the arg passed to this function here? I expected:
```suggestion
                                           physical = physical, aggregate = FALSE))
```

Reply:
Hm, seems like this was somehow in original branch, I think we shoudl change to your suggestion definitely...